data/Stephens_2023/metadata.yml

```diff
@@ -0,0 +1,106 @@
 no information was available for a particular species they were scored at genus (n = 131), or family (n = 4) level. We obtained pollination data for 1160 of 1201 taxa, using the best available evidence to score the pollination system (n = 432) or syndrome (n = 728) for each taxon. Where possible, explicit studies of pollination ecology in a taxon's native range were preferred (n = 239), especially if these involved explicit tests for the occurrence of wind pollination (45 of these). Where these data were not available, we used records of floral visitation in combination with an interpretation of species floral syndrome (n = 193). ^p Where no field observations had been recorded, we interpreted species floral syndrome sensu Faegri & van der Pijl (1979) (n = 728). Although pollination syndromes can be inaccurate at fine taxonomic levels (Ollerton et al., 2009; van der Niet, 2021), they are effective predictors of the broad pollination groups used here (Rosas-Guerrero et al., 2014; Dellinger, 2020), especially wind pollination which has a well-defined suite of traits (Friedman & Barrett, 2008). Floral syndrome was interpreted from species descriptions, illustrations and images from various sources, including iNaturalist and eFloras (eFloras, 2022; iNaturalist, 2022), and informed by pollination syndrome data from trait databases including TRY (Kattge et al., 2020), BiolFlor (Kühn et al., 2004), and AusTraits (Falster et al., 2021). Full references are available in Supporting Information. ^p Floral syndrome was scored by considering all available evidence. To separate wind from animal pollinated flowers we assessed traits in Table 1 of Friedman & Barrett (2009) and pollen as described in Hu et al. (2008), particularly perianth size and colour, gynoecium size and shape, pollen and floral rewards (Table 1). To separate insect from vertebrate pollination syndromes we considered floral size and the robustness of floral parts, nectar quantity and the accessibility of floral rewards to different pollinators (e.g. the presence of poricidal anthers which only release pollen when vibrated by bees in buzz-pollinated flowers (Pritchard & Vallejo-Marín, 2020), Table 1). Water pollination was considered in the rare cases where plants had an aquatic habit and flowered near or under water (Ackerman, 2000). ^p Flowers were scored as polymorphic where there was evidence for more than one pollination mode, or where they were pollinated by animals but it was unclear whether this was vertebrate or insect pollination (n = 76). Where there was no evidence that pollination via external vectors occurred (in clonal or autogamous species) or no information was available these species were left as missing data (n = 41). Our final dataset included pollination information for 1160 of the 1201 species in 433 families (all except Hoplestigmataceae) in the Ramírez-Barahona et al. (2020) angiosperm tree. Fully referenced data are available at https://doi.org/10.5281/zenodo.7592528.
```

Comment:
Text for methods needs to go in quotes, as text contains a `:`

Reply:
fixed